qml/components/AlbumPage.qml

```diff
@@ -41,6 +41,17 @@ Page {
         selectionText: qsTranslate("unplayer", "%n track(s) selected", String(), tracksProxyModel.selectedIndexesCount)
 
         PushUpMenu {
+            MenuItem {
+                enabled: tracksProxyModel.hasSelection
+                text: qsTranslate("unplayer", "Replace queue")
+
+                onClicked: {
+                    Unplayer.Player.queue.clear()
+                    Unplayer.Player.queue.addTracksFromLibrary(tracksModel.getTracks(tracksProxyModel.selectedSourceIndexes))
```

Comment:
`addTracksFromLibrary()` has second boolean parameter `clearQueue`, better use it instead of calling clear() manually (result will be the same though).

Reply:
Done. Thanks for the hint!